protocol/communities/persistence.go

```diff
@@ -2032,3 +2032,59 @@ func (p *Persistence) getDecryptedCommunityDescriptionByID(tx *sql.Tx, community
 		return nil, err
 	}
 }
+
+func (p *Persistence) GetCommunityRequestsToJoinRevealedAddresses(communityID []byte) (map[string][]*protobuf.RevealedAccount, error) {
```

Comment:
I think you can just use `GetCommunityRequestsToJoinWithRevealedAddresses`, unless maybe it's slower?